app/services/DynamoChannelTests.scala

```diff
@@ -25,16 +25,10 @@ object DynamoChannelTests {
   }
 }
 
-class DynamoChannelTests(stage: String) extends StrictLogging {
+class DynamoChannelTests(stage: String, client: DynamoDbClient) extends StrictLogging {
```

Comment:
the client is now passed in as a dependency, so that the tests can use a local instance